src/catalogue.ts

```diff
@@ -52,6 +52,34 @@ const getRequiredDataSets = (options: any) => {
     return { requiredDataSets: { type: "PRODUCTION" } };
 };
 
+const unwrapItemDescriptionObject = (itemDescription: any): string => {
+    if (itemDescription.uri) {
+        return itemDescription.uri;
+    }
+    if (itemDescription.expression) {
+        return itemDescription.expression;
+    }
+    throw new Error("Item description can only have expression or uri");
+};
+
+const definitionAndColumnsToItemDescriptionObjects = ({ columns, definitions }: any): any => {
```

Comment:
hmm what about just `convertToItemDescriptionObjects`, `buildItemDescriptionObjects`

`definitionAndColumnsToItemDescriptionObjects` its too long to me